### DISCUSSION TITLE
The "communication guidelines" link in bug template yields a 404 today.

### DISCUSSION THREAD

**Discussion by iago-lito** [Ulauncher/Ulauncher#1255] [General]
Hello, I have ticked the box for #1254, but the [[communication guidelines](https://github.com/Ulauncher/Ulauncher/blob/dev/CODE_OF_CONDUCT.md#general-communication-guidelines)] could not actually be read ^ ^"

**Comment by friday** [comment]
Thanks for noticing and letting us know 🙏 It's supposed to go to https://github.com/Ulauncher/Ulauncher/blob/v6/CODE_OF_CONDUCT.md

**Comment by friday** [comment]
Fixed now. Thank you again :+1:
